src/main/webapp/app/entities/flair-bi/filter/filter-date-range.component.js

```diff
@@ -219,7 +219,7 @@
                 return null;
             }
             // console.log('str to date', date, typeof date);
-            return new Date(date);
+            return Date.parse(date) ? new Date(date) : null;
```

Comment:
is there any scenario when `Date.parse(date)` won't return a result, but `new Date(date)`? Just curious if `new Date(date)` is needed at all here.

Reply:
this(Date.parse(date)) is just to check whether date is valid or not.
when select the dynamic date range and select the day . datepicker shows the calander of 1880 as you assign the "_FLAIR_INTERVAL_OPERATION(NOW()" to date field...it returns the "invalid date" during conversion